scripts/pre_commit.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
 set -e
 
-./scripts/build_proto.sh
+# start tests in parallel (with additional parallelism)
+pytest -m fast -n auto &
+
+# check bandit in parallel
+bandit -r src -ll &
+
+# run API documentation test notebooks in parallel
+./scripts/nb_test.sh && pytest examples/api --cov-fail-under 0 &
```

Comment:
Only the last command will run in background

Reply:
Do you mean on each line? The ones written left to right with && are dependant on each other so not sure I can do much about that. I tried running each line in a subshell $() but then its out of the virtualenv, any idea on how to improve this? It's already halved the run time on my machine so it's at least an improvement.

Reply:
Ahh..Ok. I thought the intention was here to run both commands in parallel.